src/AdminConsole/AdminConsole.csproj

```diff
@@ -8,6 +8,8 @@
     <PackageReference Include="Azure.Extensions.AspNetCore.DataProtection.Keys" Version="1.2.3" />
     <PackageReference Include="Azure.Identity" Version="1.11.0" />
     <PackageReference Include="Datadog.Trace" Version="2.41.0" />
+    <PackageReference Include="Fido2.Models" Version="4.0.0-beta.12" />
+    <PackageReference Include="Humanizer.Core" Version="2.14.1" />

```

Comment:
If you just need one method consider extracting it from their source code to keep our application more secure.